e3sm_diags/parameter/__init__.py

```diff
@@ -4,6 +4,7 @@
 from .core_parameter import CoreParameter
 from .diurnal_cycle_parameter import DiurnalCycleParameter
 from .enso_diags_parameter import EnsoDiagsParameter
+from .lat_lon_land_parameter import LatLonLandParameter
```

Comment:
In order to co-operate both lat-lon and lat-lon-land, I tried to use the CoreParameter for lat_lon_land (with different test path and test name), but it didn't work because there was a parameter check which doesn't allow same type of parameters (https://github.com/E3SM-Project/e3sm_diags/blob/a776ece46f9b0cb2154011fb39373445610a4590/e3sm_diags/run.py#L48-L53), not sure why this needs to be restricted, but I went ahead to added a new parameter object.

Reply:
My initial impression is that you can't have "two or more parameters of the same type" because those parameters (aka class attributes) are copied from the parent class (`CoreParameter`) to the child classes (such as `LatLonLandParameter`). A class attribute can't be defined twice, just like a variable.

`self._add_parent_attrs_to_children()` does the copying

https://github.com/E3SM-Project/e3sm_diags/blob/a776ece46f9b0cb2154011fb39373445610a4590/e3sm_diags/run.py#L48-L55

https://github.com/E3SM-Project/e3sm_diags/blob/a776ece46f9b0cb2154011fb39373445610a4590/e3sm_diags/run.py#L94-L104

Reply:
Okay, thanks, it seems there has to be a new parameter object for lat-lon-land in this framework. I will go ahead and merge.